operator/Makefile

```diff
@@ -62,12 +62,8 @@ manifests: kustomize controller-gen ## Generate WebhookConfiguration, ClusterRol
 fmt: ## Run go fmt against code.
 	go fmt ./...
 
-.PHONY: vet
-vet: ## Run go vet against code.
-	go vet ./...
-
```

Comment:
does vet not work anymore? just wondering why we're getting rid of it

Reply:
why vet in both operator and root? its the same project